mcux/README

```diff
@@ -80,3 +80,7 @@ Patch List:
    * mcux: Add support for mimxrt1050_evk_qspi. The following files were added by this commit:
       mcux/mcux-sdk/boards/evkbimxrt1050/xip/driver_xip_board_evkbimxrt1050_qspi.cmake
       mcux/mcux-sdk/boards/evkbimxrt1050/xip/evkbimxrt1050_flexspi_nor_qspi_config.c
+
+   * mcux: edma: fix issues with scatter/gather mode, and add support for FSL_FEATURE_MEMORY_HAS_ADDRESS_OFFSET
+      mcux/mcux-sdk/drivers/edma/fsl_edma.c
+      mcux/mcux-sdk/drivers/edma/fsl_edma.h
```

Comment:
I think this PR is also updating some SAI files. Can this be added as well?

Reply:
> I think this PR is also updating some SAI files. Can this be added as well?

Hi @mmahadevan108 , with PR #140 merged, I rebased this commit.  Now the readme also includes the SAI file changes.